apps/events/constants.py

```diff
@@ -38,3 +41,67 @@ class EventType:
         (ANNET, "Annet"),
         (KJELLEREN, "Realfagskjelleren"),
     )
+
+
+class RegisterStatus(IntegerChoices):
+    # Allowed status codes
+    ALLOWED_MEMBERS = 200, "Tillatt for alle medlemmer."
+    ALLOWED_GUEST = 201, "Tillatt for alle medlemmer og gjester."
+
+    # Allowed by bundle types (21x)
+    ALLOWED_FOS_RULE = 210, "Tillatt gjennom studieretning."
+    ALLOWED_GRADE_RULE = 211, "Tillatt gjennom årskull."
+    ALLOWED_GROUP_RULE = 212, "Tillatt gjennom gruppe."
+    ALLOWED_GENERIC_RULE = 213, "Tillatt gjennom regler."
+
+    # Blocked status codes (4xx)
+    BLOCKED_MEMBERS_ONLY = 400, "Dette arrangementet er kun åpent for medlemmer."
+    BLOCKED_MARK = 401, "Din påmelding er utsatt grunnet prikk."
+    BLOCKED_NOT_OPEN_YET = 402, "Påmeldingen har ikke åpnet enda."
+    BLOCKED_NO_ACCESS = (
+        403,
+        "Du har ikke tilgang til å melde deg på dette arrangementet.",
+    )
+    BLOCKED_IS_ATTENDING = 404, "Du er allerede meldt på dette arrangementet."
+    BLOCKED_SUSPENSION = 405, "Du er suspendert og kan ikke melde deg på."
+
+    # Blocked by rule (41x)
+    BLOCKED_FOS_RULE = (
+        410,
+        "Din studieretning er en annen enn de som har tilgang til dette arrangementet.",
```

Comment:
```suggestion
        "Din studieretning har ikke tilgang til dette arrangementet.",
```
To be consistent with the other messages